server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/K8sJobCaller.java

```diff
@@ -36,17 +42,26 @@ public K8sJobCaller(K8sJobClient client, PodConfig podConfig) {
 
     @Override
     public void doStart(JobContext context) throws JobException {
-        String jobName = JobUtils.generateJobName(context.getTaskId());
+        String jobName = JobUtils.generateJobName(context.getJobIdentity());
         PodParam podParam = new PodParam();
-        podParam.getEnvironments().put(JobEnvConstants.TASK_PARAMETER, JsonUtils.toJson(context));
+        Map<String, String> envs = podParam.getEnvironments();
+        envs.put(JobEnvConstants.BOOT_MODE, JobConstants.ODC_BOOT_MODE_EXECUTOR);
+        envs.put(JobEnvConstants.TASK_RUN_MODE, TaskRunModeEnum.K8S.name());
+        envs.put(JobEnvConstants.TASK_PARAMETER, JsonUtils.toJson(context));
+
+        envs.put("DATABASE_HOST", SystemUtils.getEnvOrProperty("ODC_DATABASE_HOST"));
```

Comment:
Why set database connect info to ENV? Executor no need these envs?

Reply:
because dml need meta db store some task middle data